packages/patterns-react/src/patterns/CardWithImages/README.md

```diff
@@ -0,0 +1,79 @@
+# Cards with images
+
+> The Cards with images pattern is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { CardsWithImages } from '@carbon/ibmdotcom-patterns-react';
+import 'yourapplication.scss';
+import '@carbon/ibmdotcom-styles/scss/patterns/cards-with-images/index.scss';
+
+function App() {
+  return;
+  <CardsWithImages cardsGroup={cardsGroup} />;
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the cards-with-images styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+#### Feature Flags
+
+To utilize the following features, set the following variable's to `true` within
+your `.env` file or your application build settings.
+
+```
+DDS_CARDS_WITH_IMAGES=true
+```
+
+> See
+> [feature-flags.md](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/patterns-react/docs/feature-flags.md)
+> and
+> [.env.example](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/patterns-react/.env.example)
+> for more information
+
+## cardsGroup ( Array of Objects )
+
+| Name        | Description                                                         |
+| ----------- | ------------------------------------------------------------------- |
+| `title`     | Cards with images title                                             |
+| `groupCard` | groupCard, array with href, text and target properties              |
+| `cards`     | Cards, array of objects with image, title, copy and link properties |
```

Comment:
@mkothur seems a bit vague what can be passed exactly, @annawen1 do we have an example that includes the structure of objects that are getting passed in?

Also, if there are updates to `CardLink`, then that README should get updated too.

Reply:
Okay